docker-compose.yml

```diff
@@ -20,6 +20,13 @@ services:
             - "8888:80"
         depends_on:
             - db
+        environment:
+            APP_NAME: email-service
```

Comment:
I'd prefer sticking this in a .env file with a .env-dist example file. Don't want passwords leaking into source repositories. :)

Reply:
I'm assuming you mean `API_ACCESS_KEYS: abc123` even though the comment is tagged on `APP_NAME: email-service`. It's already there, but I'll go ahead and add the `abc123` to the dist file.

Reply:
Actually, I meant all of them... even the MYSQL_ stuff below.